wsi/drm/drmdisplay.cpp

```diff
@@ -782,10 +861,29 @@ bool DrmDisplay::Commit(
     display_queue_->ResetPlanes(pset.get());
 
   if (display_state_ & kNeedsModeset) {
+    /* KK: Put to check only if input layer is a hdr */
+    for (const DisplayPlaneState &comp_plane : composition_planes) {
```

Comment:
Need to check if the layer is HDR layer, otherwise no need to apply HDR drm mode set

Reply:
also need check if the display supports HDR here, for example check display_hdrMd